content/blog/how-computers-speak/index.md

```diff
@@ -437,6 +437,8 @@ Some folks have even been able to write [React code using GPT-3.0](https://twitt
 
 Only time travelers will know precisely how AI will play out with using English for our programming in the future, but there are many obstacles to overcome before it becomes the norm.
 
-# Conclusion
+# Conclusion {#conclusion}
```

Comment:
does our MD system not automatically do this for headers?

Reply:
It does, I just like doing this for redundancy so they can be safely renamed in the future